packages/mobile/src/screens/edit-track-screen/components/HiddenAvailability.tsx

```diff
@@ -36,7 +36,7 @@ const useStyles = makeStyles(({ spacing, palette }) => ({
     alignItems: 'center'
   },
   title: {
-    fontSize: 22,
+    fontSize: 18,
```

Comment:
this could probably be deleted if we use the new `Text` component but i'm not gonna block this PR on it.

Reply:
yeah i tried at first and then i realized we're missing some variant that we were supposed to have here via harmony, so going to just wait on that for now